types/relay-compiler/lib/core/RelayCompilerError.d.ts

```diff
@@ -0,0 +1,10 @@
+import { ASTNode, GraphQLError } from 'graphql';
+import { Location } from './GraphQLIR';
+
+export type UserError = GraphQLError;
+
+export function createUserError(
+  message: string,
+  locations?: readonly Location[],
+  nodes?: readonly ASTNode[],
```

Comment:
Seeing as these are equivalent to `ReadonlyArray<…>`, I would suggest using that instead so we don’t need to bump the TS version requirement yet.

Reply:
Agree, misread the documentation so thought ReadonlyArray was also new. Changed accordingly.

Reply:
Great 👍

Reply:
About the other thing about adding typescript  to the dependencies, isn't it the same for DT as with all other packages, that dependencies are installed when you install the package eg. @types/relay-compiler? In that then you'd typically just have typescript (but also graphql as I can see is already a dependency) as dev- and peerDependencies, right? Is DT not installing these dependencies the same way?